src/commands/info.ts

```diff
@@ -0,0 +1,60 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction, EmbedBuilder, EmbedAuthorOptions,ColorResolvable, APIEmbedField } from 'discord.js';
+import * as config from "../config.json";
+import { Command } from '../types/command';
+import { PermissionLevel } from '../utils/permissions';
+
+
+const BotInfo: Command = {
+	permissionLevel: PermissionLevel.TEAM_MEMBER,
```

Comment:
I'm fine with anyone running this
```suggestion
	permissionLevel: PermissionLevel.MEMBER,
```